autofill/autofill-impl/src/test/java/com/duckduckgo/autofill/impl/service/AutofillProviderChooseViewModelTest.kt

```diff
@@ -81,6 +93,19 @@ class AutofillProviderChooseViewModelTest {
         }
     }
 
+    @Test
+    fun whenContinueAfterAuthenticationThenUpdateLastUsedTimestamp() = runTest {
```

Comment:
nit: it’s not immediately clear how “ThenUpdateLastUsedTimestamp” is verified in this test. 
- is it that `AutofillLogin` command is only sent when the last-used was updated, and so if that happens you know last-used was updated?
- or should the test be renamed?

Reply:
good catch! updating the test to validate that.